client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx

```diff
@@ -115,6 +115,16 @@ const VerticalSelect: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 		onSubmit();
 	};
 
+	const handleBlur = () => {
+		const lastQuery = inputText.trim();
+		if ( isFocused && lastQuery.length ) {
+			const vertical = suggestions.find( ( { label } ) =>
+				label.toLowerCase().includes( lastQuery )
```

Comment:
Maybe lower case `inputText` too.

Reply:
decided to use the existing suggestions for which we already did the comparison: https://github.com/Automattic/wp-calypso/pull/40645/commits/e39bd0626401d099427a4b5737b80978600528f3